utils/slice.go

```diff
@@ -0,0 +1,32 @@
+// Copyright 2022 Jetpack Technologies Inc and contributors. All rights reserved.
+// Use of this source code is governed by the license in the LICENSE file.
+
+// Package devbox creates isolated development environments.
+package utils
```

Comment:
Could we name this something like `pkgslice`?

```suggestion
// Package pkgslice manipulates slices of devbox package names.
package pkgslice
```